wp-littlefoot.php

```diff
@@ -0,0 +1,133 @@
+<?php
+/**
+ * Plugin Name:       littlefoot
+ * Description:       Footnotes without the footprint.
```

Comment:
@goblindegook This was copied from littelfoot, but we probably need to add more context to what this plugin does to the Footnotes block. Same thing for the "Plugin Name".

Reply:
The name of the library refers to its small size, it's not very useful when describing a WordPress plugin. Try this:

> Elegant, interactive popups for footnotes.

Reply:
How about:
```
 * Plugin Name:       littlefoot
 * Description:       Elegant, interactive popups for the Footnotes block.
```

@goblindegook @xipasduarte

Reply:
@goblindegook Same as you said, but referring to the Footnotes block explicitly.